sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredentialBuilder.java

```diff
@@ -43,6 +41,6 @@ public SharedTokenCacheCredentialBuilder allowUnencryptedCache(boolean allowUnen
      */
     public SharedTokenCacheCredential build() {
         return new SharedTokenCacheCredential(username, clientId, tenantId,
-                identityClientOptions.enablePersistentCache(true));
+                identityClientOptions.enablePersistentCache());
```

Comment:
We need to update the `SharedTokenCacheCredential` to also take an `AuthenticationRecord`. See https://github.com/Azure/azure-sdk-for-net/pull/13612. If you'd like we can merge this first and add this in a second PR, but we need to add it before the release.